src/classes/GE_FormRendererService.cls

```diff
@@ -106,6 +111,45 @@ public with sharing class GE_FormRendererService {
 
     }
 
+    private static String convertWidgetDataToObjectJSON(String widgetData) {
+        // System.debug('widgetData: '); 
+        // System.debug(widgetData); 
+
+        List<SObject> thisList;
+        Map<String, Object> objNameToObjList = (Map<String, Object>)JSON.deserializeUntyped(widgetData);
```

Comment:
We need to handle the case where deserialization fails here, and check for null before attempting to deserialize.